sycl/test/abi/sycl_symbols_windows.dump

```diff
@@ -341,7 +341,6 @@
 ??0half_v2@host_half_impl@detail@sycl@cl@@QEAA@AEBM@Z
 ??0half_v2@host_half_impl@detail@sycl@cl@@QEAA@AEBV01234@@Z
 ??0half_v2@host_half_impl@detail@sycl@cl@@QEAA@G@Z
-??0half_v2@host_half_impl@detail@sycl@cl@@QEAA@XZ
```

Comment:
@alexbatashev, is this considered to be an ABI break?

Reply:
This line was introduced in https://github.com/intel/llvm/pull/4518 . It is not in the latest release